pkg/operator/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -112,27 +112,18 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		&workqueue.BucketRateLimiter{Limiter: rate.NewLimiter(rate.Limit(10), 100)},
 	)
 
-	// Create controller with dependencies set
-	c := &internalcontroller.Controller{
-		Do:                      r,
-		Cache:                   mgr.GetCache(),
-		Config:                  mgr.GetConfig(),
-		Scheme:                  mgr.GetScheme(),
-		Client:                  mgr.GetClient(),
-		Recorder:                mgr.GetEventRecorderFor(name),
-		Queue:                   workqueue.NewNamedRateLimitingQueue(rateLimiter, name),
-		MaxConcurrentReconciles: 1,
-		Name:                    name,
-	}
-
-	if err := mgr.Add(c); err != nil {
```

Comment:
Needed to remove this mgr.Add() as it is already called in the controller.New() codepath right above. Without this CCO errors early on startup with `time="2021-11-01T16:28:09Z" level=fatal msg="unable to run the manager" error="controller was started more than once. This is likely to be caused by being added to a manager multiple times"`